runtime/basilisk/src/lib.rs

```diff
@@ -170,13 +171,9 @@ construct_runtime!(
 		Treasury: pallet_treasury = 4,
 		Utility: pallet_utility = 5,
 		//NOTE: 6 - is used by Scheduler which must be after cumulus_pallet_parachain_system
-		Democracy: pallet_democracy exclude_parts { Config } = 7,
```

Comment:
we should probably retire those pallet id's of removed pallets

Reply:
wdym by retire the pallet IDs?

Reply:
not use them in future